CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## Fleet 4.48.3 (Apr 16, 2024)
+
+### Bug fixes
+
+* Calendar webhook will retry if it receives response 429 Too Many Requests. Webhook request will retry for 30 minutes with a 1 minute max delay between retries.
```

Comment:
All changelog entries should be phrased in the past tense. ChatGPT gets most of them right, but manually edits are usually required. Example: 

"Updated calendar webhook to retry if..."